airbyte-integrations/connectors/destination-redshift/src/main/java/io/airbyte/integrations/destination/redshift/RedshiftDestination.java

```diff
@@ -0,0 +1,132 @@
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.airbyte.integrations.destination.redshift;
+
+import static org.jooq.impl.DSL.field;
+import static org.jooq.impl.DSL.table;
+import static org.jooq.impl.DSL.unquotedName;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.base.Charsets;
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.lang.CloseableQueue;
+import io.airbyte.integrations.base.Destination;
+import io.airbyte.integrations.base.IntegrationRunner;
+import io.airbyte.integrations.destination.jdbc.AbstractJdbcDestination;
+import io.airbyte.protocol.models.AirbyteRecordMessage;
+import java.sql.SQLException;
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.UUID;
+import org.jooq.DSLContext;
+import org.jooq.InsertValuesStep3;
+import org.jooq.Record;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RedshiftDestination extends AbstractJdbcDestination implements Destination {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(RedshiftDestination.class);
+  static final String COLUMN_NAME = AbstractJdbcDestination.COLUMN_NAME;
+
+  public RedshiftDestination() {
+    super("com.amazon.redshift.jdbc.Driver", null, new RedshiftSQLNaming());
+  }
+
+  @Override
+  public JsonNode toJdbcConfig(JsonNode redshiftConfig) {
+    return Jsons.jsonNode(ImmutableMap.builder()
+        .put("username", redshiftConfig.get("username").asText())
+        .put("password", redshiftConfig.get("password").asText())
+        .put("jdbc_url", String.format("jdbc:redshift://%s:%s/%s",
+            redshiftConfig.get("host").asText(),
+            redshiftConfig.get("port").asText(),
+            redshiftConfig.get("database").asText()))
+        .put("schema", getDefaultSchemaName(redshiftConfig))
+        .build());
+  }
+
+  @Override
+  public String createRawTableQuery(String schemaName, String streamName) {
+    final String result = String.format(
+        "CREATE TABLE IF NOT EXISTS %s.%s ( \n"
+            + "ab_id VARCHAR PRIMARY KEY,\n"
+            + "%s VARCHAR(max),\n"
+            + "emitted_at TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP\n"
+            + ");\n",
+        schemaName, streamName, COLUMN_NAME);
+    return result;
+  }
+
+  @Override
+  public void writeBufferedRecords(int batchSize, CloseableQueue<byte[]> writeBuffer, String schemaName, String tmpTableName) {
```

Comment:
why does redshift need to override this? cant it just used what's already in AbstractJdbcDestination?